### ISSUE TITLE
Settings structure not optimal for VoiceOver

### ISSUE THREAD

**Issue by isidorn** [Microsoft/vscode#72079]
From @PeterSaathoff-Harshfield

Settings editor: tab around after the commonly used comes, "settings list focus helper, group," which seems to outline most of the window and contains no other elements.
Next is "settings, table." This appears to contain the bulk of the settings. Interacting with this area does nothing. VoiceOver knows about no elements within. Interestingly, using tab at this point will cause VoiceOver to move focus to controls within this area, but only editable controls and links. None of the static text is accessible.
Below the main settings area is another "settings list focus helper" that appears to do nothing.
At the bottom of the window is a bar that VoiceOver reports as, "content info." The elements in this bar are navigated to by VoiceOver in the opposite order than they appear

**Comment by roblourens** [comment]
The static text should be read when you focus the setting control. I'll have to investigate

**Comment by roblourens** [comment]
> At the bottom of the window is a bar that VoiceOver reports as, "content info." The elements in this bar are navigated to by VoiceOver in the opposite order than they appear

I'm not sure what you're referring to

**Comment by PeterSaathoff-Harshfield** [comment]
I made a short video demonstrating this "right-to-left" order. https://youtu.be/SPHmXH4OY6E

**Comment by roblourens** [comment]
Ok, that is called the status bar, it is not related to the settings editor.

**Comment by isidorn** [comment]
Afaik we do not allow focus to go to the status bar.
Aslo assigning to @bpasero

**Comment by bpasero** [comment]
We intentionally do not allow to focus into the status bar so that we reduce the number of tab stops in the UI and not frustrate the user that needs to use Tab to navigate. I do not think we should chang this unless people that use Tab key for navigation ask for this.

**Comment by PeterSaathoff-Harshfield** [comment]
I was not indicating that there was anything happening with tabbing through the status bar.
What I am referring to is that, using standard VoiceOver navigation commands (control + option + left/right arrow), the elements in the status bar are in the opposite order than they appear visually.

**Comment by bpasero** [comment]
@PeterSaathoff-Harshfield ah yes good point, this is due to the use of `float: right` down there which I might want to revisit 👍

**Comment by isidorn** [comment]
@bpasero We are now using flex down there
@PeterSaathoff-Harshfield the status bar behavior should be better

**Comment by bpasero** [comment]
@isidorn ah yeah, forgot to come back here. the order of items is still reverse though, even when using flex. I need to do that so that elements wrap in an order that is good for us (e.g. the notification icon should drop last).

If someone has an idea for this question I can change that: 

I have a div with `display: flex` and multiple elements within. When I shrink it’s size, I want the first item to hide (=wrap) but not the last. By default, `flex` starts to wrap the last element which makes sense. I know I can do it via `flex-direction: row-reverse` but that makes it complicated because elements in the DOM have to be in reverse order.

**Comment by isidorn** [comment]
Let's add help-wanted so maybe some nice soul comes and answers your question.

**Comment by bpasero** [comment]
Should we move this to a new issue though? I find the summary has nothing to do with status bar.

**Comment by isidorn** [comment]
Yeah, created https://github.com/microsoft/vscode/issues/76010

**Comment by isidorn** [comment]
@roblourens I see there are quite some issues settings UX not accessibilie. Maybe we compile all the data in one issue and try to close duplicates?
